### PR TITLE
Travis CI integration

### PR DESCRIPTION
**Pull request by azakus**
Needs some work probably

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    sources:
+    - google-chrome
+    packages:
+    - google-chrome-beta
+cache: yarn
+before_script:
+- mkdir -p ~/bin
+- ln -s /usr/bin/google-chrome-beta ~/bin/google-chrome
+- export PATH=$HOME/bin:$PATH
+- npm build
```
Comment (TimvdLippe) [inline .travis.yml]: This should be `npm run build`

Comment (justinfagnani) [inline .travis.yml]: fixed

**Comment by justinfagnani** [comment]
@azakus have you seen this error on Travis?

> Error response status: 6 Selenium error: no such session

https://travis-ci.org/PolymerLabs/lit-html

**Comment by azakus** [comment]
HA, forgot to turn on the display server...

**Comment by justinfagnani** [comment]
Thanks @azakus !

**Pull request merged**